### ISSUE TITLE
Phase.from_cif() fails because space_group cannot be read

### ISSUE THREAD

**Issue by OverEthanol** [pyxem/orix#505]
When reading `.cif` file from a folder outside of the current directory using, for example

```Python
from pathlib import Path
from orix.crystal_map import Phase
dataDir = Path("C:/Users/s175423/Experiment_Data")
structure = Phase.from_cif(dataDir / "xxx.cif")
```

It outputs the error message that:

![image](https://github.com/user-attachments/assets/37be7250-6263-45e6-8481-18ed411479c0)

However, when I load in the `.cif` file from the working directory, then everything is fine again. Is this perhaps a bug?

**Comment by pc494** [comment]
Hi,

This is certainly very weird. Would it be possible to share a copy of the `.cif` file in question?

**Comment by OverEthanol** [comment]
> Hi,
> 
> This is certainly very weird. Would it be possible to share a copy of the `.cif` file in question?

Try and see if you can download this one __[`ZIF-L.cif`](https://dtudk-my.sharepoint.com/:u:/g/personal/yuxzh_dtu_dk/Ec5t_3w_v6JAuvDU1bAJ_ZcBAGYfZ3dCC1ft-_-Z5OJEiw?email=absf%40dtu.dk&e=KwhYHN)__

**Comment by pc494** [comment]
> > Hi,
> > This is certainly very weird. Would it be possible to share a copy of the `.cif` file in question?
> 
> Try and see if you can download this one **[`ZIF-L.cif`](https://dtudk-my.sharepoint.com/:u:/g/personal/yuxzh_dtu_dk/Ec5t_3w_v6JAuvDU1bAJ_ZcBAGYfZ3dCC1ft-_-Z5OJEiw?email=absf%40dtu.dk&e=KwhYHN)**

This file is protected to your institution.

**Comment by OverEthanol** [comment]
> This file is protected to your institution.

Very sorry about that! Can you try [this](https://github.com/user-attachments/files/16361617/ZIF-L_1509273.zip) instead?

**Comment by hakonanes** [comment]
Thanks for reporting this, @OverEthanol. The space group is not required for a functional `Phase` instance, so we should of course just *try* to get the space group number and then move on if we couldn't read it... I'll have a look at this.

**Comment by hakonanes** [comment]
A fix is available in #522.
